pyproject.toml

```diff
@@ -243,7 +243,9 @@ exclude = [
     "venv",
     "tests/assets",
 
-    "src/otx/core/engine/utils/*"   # It will be clean up later
+    # it will be cleaned up later
+    "src/otx/core/engine/utils/*",
+    "src/otx/algo/*",
```

Comment:
I'm not sure this is the right way to go.
Will this be done all at once at a later date, which could cause us a lot of pain down the road?
As @jaegukhyun said, this could be a big issue later.